CMMS/Graph/Entry/WOOrderEntry.cs

```diff
@@ -18,25 +22,29 @@ public class WOOrderEntry : PXGraph<WOOrderEntry, WOOrder>
             .View CurrentDocument;
 
         [PXViewName(Messages.ViewTransactions)]
+        [PXImport(typeof(WOLine))]
```

Comment:
I might be totally mistaken, but dont these need to be PXImport(typeof(MainDAC)) ? so [PXImport(typeof(WOOrder))]

Reply:
Resolved in latest commit